cirq-core/cirq/work/sampler.py

```diff
@@ -197,7 +199,7 @@ async def run_sweep_async(
 
     def run_batch(
         self,
-        programs: List['cirq.Circuit'],
+        programs: List['cirq.AbstractCircuit'],
```

Comment:
I think this should be `Sequence['cirq.Abstractcircuit']`. Sequence is covariant whereas list is not

https://mypy.readthedocs.io/en/stable/generics.html#variance-of-generics